src/copy.c

```diff
@@ -298,9 +828,25 @@ copyfrom(CopyChunkState *ccstate, List *range_table, Hypertable *ht, void (*call
 
 		CHECK_FOR_INTERRUPTS();
 
-		/* Reset the per-tuple exprcontext */
+		/*
+		 * According to the setting 'timescaledb.max_open_chunks_per_insert',
```

Comment:
As an aside, the way we calculate the default value for that setting looks suspicious. Might have to revisit that in the future.